app/controllers/rounds_controller.rb

```diff
@@ -40,9 +34,70 @@ def update
     end
   end
 
+  def activate
+    if @round.valid?
+      if @round.active == false
+        @round.update(active: true)
+        redirect_to @game, notice: "Round #{@round.category} actived"
+      else
+        redirect_to @game, notice: "Round #{@round.category} is already active"
+      end
+    else
+      redirect_to @game, notice: "Game or round not found..."
+    end
+  end
+
+  def deactivate
+    if @round.valid?
+      if @round.active == true
+        @round.update(active: false)
+        redirect_to @game, notice: "Round #{@round.category} deactivated"
+      else
+        redirect_to @game, notice: "Round #{@round.category} has already been deactivated"
+      end
+    else
+      redirect_to @game, notice: "Game or round not found..."
+    end
+  end
+
+  def complete
+    if @round.valid?
+      if @round.completed == false
+        @round.update(completed: true)
+        redirect_to @game, notice: "Round #{@round.category} completed"
+      else
+        redirect_to @game, notice: "Round #{@round.category} has already been set to complete"
+      end
+    else
+      redirect_to @game, notice: "Game or round not found..."
+    end
+  end
+
+  def continue
+    if @round.valid?
+      if @round.completed == true
```

Comment:
Same as above for complete.
```suggestion
      if @round.completed
```